great-idea/services.html

```diff
@@ -15,93 +16,117 @@
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
-
-
-
-iOS Development
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
-
-
-
-Android Development
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
-
-
-Some Facts About Our Services
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-
-Awesome thing
-Amazing thing
-Cool thing
-Great thing
-
-<img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
-
-Copyright Great Idea! 2018
-
+  <header>
+    <div class="container">
+      <div class="header-inner">
+        <nav>
+          <ul>
+            <li><a href="#services">Services</a></li>
+            <li><a href="products.html">Product</a></li>
+            <li><a href="#vision">Vision</a></li>
+            <li><a href="#features">Features</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+        <div id="logo">
+          <div class="image-container">
+            <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <div class="container">
+    <section id="banner">
+      <div class="image-container">
+        <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+      </div>
+    </section>
+    <hr>
+    <section class="company-business">
+      <div class="services">
+        <h3>Services</h3>
+        <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus
+          rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur
+          cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod
+          facilisis iaculis.</p>
+      </div>
+      <div class="business-item-container">
+        <article class="company-business-item">
```

Comment:
`<article>` should only be used for specific content like news article or comments - https://www.w3schools.com/tags/tag_article.asp

Reply:
Okay. Thanks for the feedback.